lib/bump.rb

```diff
@@ -57,6 +58,21 @@ def current
 
       private
 
+      def sanitize_options!(options)
+        options.each do |key, value|
+          options[key] = eval_true_false_and_nil(value)
+        end
+        options.delete_if{|key, value| value.nil?}
+      end
+
+      def eval_true_false_and_nil(value)
+        if ["true", "false", "nil"].include? value
+          eval(value)
```

Comment:
this is bit scary ... how about `case foo; when 'true' then true; when 'false' then false` etc

Reply:
sure @grosser, if you think this is scary, I will remove `eval`.